app/controllers/admin/users/index.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
```

Comment:
@harshitagupta30 We do not need an index controller, please make use of dynamic routes for all users also.